terraform/ecs/analytics.tf

```diff
@@ -0,0 +1,76 @@
+resource "aws_s3_bucket" "analytics_data_lake-bucket" {
+  bucket = var.analytics-data-lake_bucket_name
+}
```

Comment:
@Rakowskiii  still not sure, is this creating a new bucket?  from the output of the terraform diff looks like we're still creating one.